cob_hardware_config/raw3-5/config/base_controller.yaml

```diff
@@ -20,17 +21,21 @@ twist_controller:
       spring: 15.0 #10.0
       damp: 2.75 #2.5
       virt_mass: 0.1 #0.1
-      d_phi_max: 12.0 #100.0
+      d_phi_max: 10.0 #100.0
       dd_phi_max: 40.0 #100.0
   wheels: &wheels
     - steer: fl_caster_rotation_joint
       drive: fl_caster_r_wheel_joint
+      steer_neutral_position: -180.0
```

Comment:
When the neutral position was set to 0 degrees the 3 wheels did move back when the command was moving forward. Changing the neutral position to -180 degrees, the wheels moved 180 degrees and  move forward when the command from the joystick was to move forward. If it does not depend on the neutral position, should we change the encoder values?